lib/grocer/ssl_connection.rb

```diff
@@ -22,6 +22,7 @@ def connected?
 
     def connect
       context = OpenSSL::SSL::SSLContext.new
+      context.security_level = 1
```

Comment:
It might be worthwhile adding a test for this.

Reply:
I thought about it. But I can't seem to be able to run their RSpec test suite locally🙈